include/pmacc/PMaccConfig.cmake

```diff
@@ -322,6 +322,25 @@ if( ("${ALPAKA_CUDA_COMPILER}" STREQUAL "nvcc") AND
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DBOOST_NO_CXX11_NOEXCEPT")
 endif()
 
+# GCC's C++11 tuples are broken in "supported" NVCC versions
+if("${ALPAKA_CUDA_COMPILER}" STREQUAL "nvcc")
+    if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
+        if(CUDA_VERSION VERSION_EQUAL 8.0)
+            if(CMAKE_CXX_COMPILER_VERSION GREATER_EQUAL 5.0)
```

Comment:
This is wrong: it must because the tuple issue was introduced with gcc 5.4

```
if(CMAKE_CXX_COMPILER_VERSION GREATER_EQUAL 5.4)
    message(FATAL_ERROR "NVCC 8.0 does not support the std::tuple "
    "implementation in GCC 5.4+. Please use GCC 4.9 - 5.3!")
endif()
```

Reply:
I will create a new PR soon to fix this. I like to keep my gcc5.3 chain ;-)

Reply:
opened #2636 to fix it

Reply:
ok, good to know! thx!